parse_and_import_medicover.py

```
@@ -170,27 +170,54 @@ def main(
+            structure = 'nested'
+        else:
             print(f"Skipping {report} as it doesn't have any data")
             skipped_reports += 1
             continue
 
-        evaluations = utils.get_evaluations(report_data)
+        print(f"Processing report: {report}")
 
-        for j, evaluation in enumerate(evaluations, 1):
+        for j, evaluation in enumerate(evaluations, startPoint):
             report_evaluation = f"{Path(report).stem}-{j}"
```

Ruff (0.14.6)

Local variable `report_evaluation` is assigned to but never used

Remove assignment to unused variable `report_evaluation`

(F841)